### ISSUE TITLE
Angular 16 support

### ISSUE THREAD

**Issue by guillermodelagala** [webcat12345/ngx-intl-tel-input#495]
Please, make it compatible with the latest Angular 16 version.

**Comment by Mozaritta** [comment]
Is it compatible with the 15 version? I have a problem using it in my project.

**Comment by hezyz** [comment]
had the same problem so modified for angular 16 and removed ngx-bootstrap, I use bootstrap 5.3 
[ngx-bs-tel-input](https://github.com/hezyz/ngx-bs-tel-input)

**Comment by edgarinoa** [comment]
> 

You haven't published it on npm?

**Comment by hezyz** [comment]
Sorry not yet 
U can pack it, and install it as tgz file

**Comment by prasantht96** [comment]
> > @hezyz
> 
> You haven't published it on npm?

https://www.npmjs.com/package/ngx-bs-tel-input

**Comment by hezyz** [comment]
> 

I did few days ago

**Comment by edgarinoa** [comment]
@hezyz yes, I noticed recently, I'm already using it in my project, thanks.

**Comment by Esmail51** [comment]
it' not working when i try to import it show's the module is not there

**Comment by Magesvarane-S** [comment]
For Angular 16, you can use this: https://www.npmjs.com/package/ngx-intl-tel-input-gg
with "ngx-bootstrap": "^9.0.0",

**Comment by webcat12345** [comment]
Added @guillermodelagala to collaborator to repository and npm package as well.

**Comment by mehulk05** [comment]
You can use this one 
https://www.npmjs.com/package/ngx-international-tel-input
